the-blue-alliance-ios/TBAAllianceCell.m

```diff
@@ -32,10 +32,18 @@ - (void)setEventAlliance:(EventAlliance *)eventAlliance {
     }
     
     // OH PICK BOY http://photos.prnewswire.com/prnvar/20140130/NY56077
+    NSInteger pickNumber = 0;
     for (Team *team in eventAlliance.picks) {
```

Comment:
Could we make this either a basic for(int i = 0;... sort of deal, or do [eventAlliance.picks indexOfObject:team] to get the pickNumber inside of the loop?